dashboard/apps/prototype/management/commands/cache.py

```diff
@@ -25,6 +26,7 @@ def generate(self):
         """
         generate cache
         """
+        cache_tools.METHOD_CACHE_WRITE_ONLY = True
```

Comment:
ok, this actually sounds a good solution. i can go with this. it's more explicit.